Content.Server/AlertLevel/ChangeAlertLevelSystem.cs

```diff
@@ -0,0 +1,58 @@
+using Content.Shared.Interaction;
+using Content.Shared.ChangeAlertLevel;
+using Content.Server.AlertLevel;
+using Content.Server.Audio;
+using Content.Server.Chat.Systems;
+using Content.Server.Explosion.EntitySystems;
+using Content.Server.Pinpointer;
+using Content.Server.Popups;
+using Content.Server.Station.Systems;
+using Content.Shared.Audio;
+using Content.Shared.Containers.ItemSlots;
+using Content.Shared.Coordinates.Helpers;
+using Content.Shared.DoAfter;
+using Content.Shared.Examine;
+using Content.Shared.Maps;
+using Content.Shared.Nuke;
+using Content.Shared.Popups;
+using Robust.Server.GameObjects;
+using Robust.Shared.Audio;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Containers;
+using Robust.Shared.Map;
+using Robust.Shared.Player;
+using Robust.Shared.Random;
+
+
+namespace Content.Server.ChangeAlertLevel;
+
+public sealed class ChangeAlertLevelSystem : EntitySystem
+{
+    [Dependency] private readonly SharedTransformSystem _transform = default!;
+    [Dependency] private readonly AlertLevelSystem _alertLevel = default!;
+    [Dependency] private readonly StationSystem _station = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<ChangeAlertLevelComponent, ActivateInWorldEvent>(OnActivated);
+    }
+
+    public void OnActivated(EntityUid uid, ChangeAlertLevelComponent comp, ActivateInWorldEvent args)
```

Comment:
Use `Entity<TComp>` all for new functions.

Reply:
i'm not entirely clear what this means, can you explain?

Reply:
https://discord.com/channels/310555209753690112/928056457117696020/1164654158725525575

Reply:
i keep getting a 'method group' error and the only working thing - which visual studio even suggests - seems to be changing back to not using Entity<T>

Reply:
have to add `ref` before the event type too

Reply:
> have to add `ref` before the event type too

can you go into more detail on this? i cannot for the life of me get any combination of things here to work and i'm having trouble finding any examples in the codebase set up like this to work from, everythings done the old way

Reply:
...nevermind it was because i did the other thing but forgot to update to using the shared version, lol. ref worked